### ISSUE TITLE
[semantic-issue] Add semantics for sklearn SVC

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#169]
**Description**
Creates an instance of the Support Vector Classification (SVC) class.
Returns an object of SVC type.

**Minimal running example**
```
from sklearn.svm import SVC

model = SVC()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html

**Note (optional)**
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py
